pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -304,6 +305,15 @@ def make_package(args):
                 tar_dirs.append(python_bundle_dir)
         if get_bootstrap_name() == "webview":
             tar_dirs.append('webview_includes')
+        if hasattr(args, "assets") and args.assets is not None:
```

Comment:
I think argparse should make this simpler, isn't it an empty list by default or something so you can just do `for asset in args.assets:`?

Reply:
I thought I tried that and it didn't work. Maybe I misconfigured argparse.

Reply:
This is when we should coverage test this part as well. Not that I'm demanding it now, but I wish we had that already covered